src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.services.AreaCalculator;
+import core.basesyntax.services.ColorSupplier;
+import core.basesyntax.services.Drawer;
+
+public class Circle extends Figure implements AreaCalculator, Drawer {

```

Comment:
```suggestion
public class Circle extends Figure {

```

if class Figure implements Drawer and AreaCalculator than all it's child classes implements Drawer and AreaCalculator